decentraland/unity-explorer
# Chore: re-enabled errors analytics

## What does this PR change?

Adds error messages being send to Analytics

- also adjusted some other schema's

## How to test the changes?

1. Launch the explorer
2. Check that "happy path" is not broken

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
